js/darkmode.js

```diff
@@ -0,0 +1,45 @@
+onload = onload();
+const icon = document.getElementById('icon');
+icon.onclick = function () {
+  const element = document.body;
+  element.classList.toggle('dark-theme');
+  if (element.classList.contains('dark-theme')) {
+    icon.src = 'img/sun.png';
+    setCookie('theme', 'dark', 30);
```

Comment:
we have decided to use `theme` as name of cookie, so could we just directly mention it In document.cookie and not over here.

Reply:
I guess we should make the function for setting the cookie reusable so that if we further need we can directly use it, @shubham-y is also doing the same.

Reply:
Great. I will do the same in my PR.